tgui/packages/tgui/interfaces/PreferencesMenu/BackgroundPage.tsx

```diff
@@ -0,0 +1,195 @@
+              disabled={characterDesc === flavor_text}
+              onClick={() => act('flavor_text', { characterDesc })}>
+              Save
+            </Button>
+            <Button icon="times" onClick={() => setCharacterDesc(flavor_text)}>
+              Reset
+            </Button>
+          </Box>
+        }>
+        <TextArea
+          key="character"
+          height="100px"
+          value={characterDesc}
+          onChange={(e, value) => setCharacterDesc(value)}
+        />
+      </Section>
+
+      <Grid>
+        <Grid.Column>
+          <Section
+            title="Medical Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={medicalDesc === med_record}
+                  onClick={() => act('med_record', { medicalDesc })}>
+                  Save
+                </Button>
+                <Button icon="times" onClick={() => setMedicalDesc(med_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={medicalDesc}
+              onChange={(e, value) => setMedicalDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+        <Grid.Column>
+          <Section
+            title="Employment Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={employmentDesc === gen_record}
+                  onClick={() => act('gen_record', { employmentDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setEmploymentDesc(gen_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={employmentDesc}
+              onChange={(e, value) => setEmploymentDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+      </Grid>
+      <Grid>
+        <Grid.Column>
+          <Section
+            title="Security Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={securityDesc === sec_record}
+                  onClick={() => act('sec_record', { securityDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setSecurityDesc(sec_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={securityDesc}
+              onChange={(e, value) => setSecurityDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
```

Comment:
Grid is deprecated in TGUI you should try to replace it with an alternative like Stack

Reply:
Will look into it